app/services/html_sanitizer.rb

```diff
@@ -6,10 +6,15 @@
 # We offer an editor which supports certain tags but you can't insert just any
 # HTML, which would be dangerous.
 class HtmlSanitizer
+  ALLOWED_TAGS = %w[h1 h2 h3 h4 div p br b i u a strong em del pre blockquote ul ol li hr
```

Comment:
It's nice and clear what's allowed now 👍 
Let's also add a comment to explain why:

```suggestion
  # div is required by Trix editor
  ALLOWED_TAGS = %w[h1 h2 h3 h4 div p br b i u a strong em del pre blockquote ul ol li hr
```